utils/jsonSigning.js

```diff
@@ -41,8 +55,48 @@ module.exports.sign = function (json, prvKey, pubKey) {
   return signedJSON
 }
 
+// Sign JSON object using RSA
+function signRSA (json, prvKey) {
+  if (!prvKey) throw new ServerError('Problem reading private key for JSON signing')
+  let prvKeyObj = null
+  // If given private key is PEM formatted string, convert it to private key object
+  // If it's already private key object, just use it
+  if (typeof prvKey === 'string') {
+    prvKeyObj = rsasign.KEYUTIL.getKey(prvKey)
+  } else {
+    prvKeyObj = prvKey
+  }
+  const strJWS = KJUR.jws.JWS.sign('PS256', // eslint-disable-line no-undef
```

Comment:
Is there a reason not to use the `PS256` variable declared at the top of the file?